index.js

```diff
@@ -1,1496 +0,0 @@
-const { exit } = require('process');
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-function getRandomInt(max) {
-  return Math.floor(Math.random() * Math.floor(max));
-}
```

Comment:
This function never gets used through your code. Using this would be the first step to reducing your code by 90% and to make the guesses programmatically